operator/controllers/keptnevaluation/providers/prometheus.go

```
@@ -0,0 +1,60 @@
+package providers
+
+import (
+	"context"
+	"fmt" //nolint:gci
+	"github.com/go-logr/logr"
+	klcv1alpha1 "github.com/keptn/lifecycle-toolkit/operator/api/v1alpha1"
+	promapi "github.com/prometheus/client_golang/api"
+	prometheus "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+	"net/http" //nolint:gci
+	"time"
+)
+
+type KeptnPrometheusProvider struct {
+	Log    logr.Logger
+	client http.Client
+}
+
+func (r *KeptnPrometheusProvider) EvaluateQuery(ctx context.Context, objective klcv1alpha1.Objective, provider klcv1alpha1.KeptnEvaluationProvider) (string, error) {
+	ctx, cancel := context.WithTimeout(ctx, 20*time.Second)
+	defer cancel()
+
+	queryTime := time.Now().UTC()
+	r.Log.Info("Running query: /api/v1/query?query=" + objective.Query + "&time=" + queryTime.String())
+	client, _ := promapi.NewClient(promapi.Config{Address: provider.Spec.TargetServer, Client: &r.client})
```

why do we ignore the error here ?